### ISSUE TITLE
PydestException: Could not connect to Bungie.net

### ISSUE THREAD

**Issue by whitew0lf5** [jgayfer/pydest#12]
```
Can not load response cookies: Illegal key 'some_key'
Ignoring exception in command test
Traceback (most recent call last):
  File "X:\Program Files\Python36\lib\site-packages\pydest\api.py", line 33, in _get_request
    json_res = await r.json()
  File "X:\Program Files\Python36\lib\site-packages\aiohttp\client_reqrep.py", line 730, in json
    headers=self.headers)
aiohttp.client_exceptions.ClientResponseError: 0, message='Attempt to decode JSON with unexpected mimetype: text/html; charset=utf-8'
```

Hey there, I'm really excited to start using this rapper for a discord bot I'm making. I was poking around with some of the code to see how it works, but every time I try to retrieve something from the destiny servers I get the above error. For example I have the following code:

```
    async def test(self, *args):
        if len(args) > 0:
            gamertag = self.format_command_arguments(args)
        else:
            return
        item = await self.dest.api.search_destiny_player(1, gamertag)
        print(item)
```

(format_command_arguments simply takes discord command arguments and makes them lowercase while concatenating them in to one string)

Sorry if this doesn't belong here, but I'm a little new to public github etiquette!

**Comment by jgayfer** [comment]
Stand by, I am taking a look.

**Comment by jgayfer** [comment]
@whitew0lf5 Can you pull the latest, re-install, and try again?

**Comment by whitew0lf5** [comment]
```
Can not load response cookies: Illegal key 'some_key'
Ignoring exception in command test
Traceback (most recent call last):
  File "X:\Program Files\Python36\lib\site-packages\pydest\api.py", line 33, in _get_request
    json_res = await r.json()
  File "X:\Program Files\Python36\lib\site-packages\aiohttp\client_reqrep.py", line 730, in json
    headers=self.headers)
aiohttp.client_exceptions.ClientResponseError: 0, message='Attempt to decode JSON with unexpected mimetype: text/html; charset=utf-8'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "X:\Program Files\Python36\lib\site-packages\discord\ext\commands\core.py", line 50, in wrapped
    ret = yield from coro(*args, **kwargs)
  File "X:\workspace\python\nerd-bot\DestinyCogs\DestinyMain.py", line 27, in test
    item = await self.dest.api.search_destiny_player(1, entity)
  File "X:\Program Files\Python36\lib\site-packages\pydest\api.py", line 116, in search_destiny_player
    return await self._get_request(url)
  File "X:\Program Files\Python36\lib\site-packages\pydest\api.py", line 35, in _get_request
    raise pydest.PydestException("Could not connect to Bungie.net")
pydest.pydest.PydestException: Could not connect to Bungie.net

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "X:\Program Files\Python36\lib\site-packages\discord\ext\commands\bot.py", line 846, in process_commands
    yield from command.invoke(ctx)
  File "X:\Program Files\Python36\lib\site-packages\discord\ext\commands\core.py", line 374, in invoke
    yield from injected(*ctx.args, **ctx.kwargs)
  File "X:\Program Files\Python36\lib\site-packages\discord\ext\commands\core.py", line 54, in wrapped
    raise CommandInvokeError(e) from e
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: PydestException: Could not connect to Bungie.net
```

**Comment by jgayfer** [comment]
Let’s try and isolate the issue to a piece of code that we know works.

Can you run the `examples/find_user.py` example in this project? Make sure that you add your API key where it says `”api-key”`.

**Comment by whitew0lf5** [comment]
find_user worked

**Comment by jgayfer** [comment]
Great!

The code snippet you sent looks fine, assuming the client was initialized with an API key :+1:

**Comment by whitew0lf5** [comment]
Ah dude I'm sorry about all this man. I fixed the issue. The error was a bit misleading because it sounded like the destiny servers were responding with weird json formats, but I figured out that the gamertag I was sending was being altered in another part of my code so the servers just simply didn't know who the user was and returned nothing. Thanks for going out of your way to help me with this though!
